M2/cmake/check-libraries.cmake

```diff
@@ -58,6 +58,7 @@ check_include_files(boost/math/tools/atomic.hpp
 # TODO: replace gdbm, see https://github.com/Macaulay2/M2/issues/594
 find_package(GDBM	REQUIRED QUIET) # See FindGDBM.cmake
 
+list(APPEND CMAKE_PREFIX_PATH "$(brew --prefix libomp)")
```

Comment:
I didn't read what you're doing in the autotools build but this should certainly not be hardcoded. It's up to the user to add brew directories to the path or not, and further there's a reason keg-only bottles are keg-only! e.g this may break depending on whether you build with GCC or clang.

Reply:
My goal was to make it simpler for the user with a reasonable set of default search paths so that the build should work out of the box most of the time on common systems without fiddling with flags.  And we're already hardcoding several other brew paths in the cmake build: https://github.com/Macaulay2/M2/commit/50ea57438295eba391c876da60be83a3e7cbe3ce

Reply:
Side note -- do we even need to deal with OpenMP on most builds?  According to a comment in configure.ac:

> This is required for building the library csdp and good for building the library normaliz

Most of the time, we'll be using existing system packages for these.